app/views/departments/index.html.slim

```diff
@@ -0,0 +1,27 @@
+.row
+  .col-xs-12
+    h1= t_title
+
+.row
+  .col-xs-12
+    table.table.table-striped
+      thead
+        tr
+          th= t_attr(:name)
+          th= t_attr(:street, Contact)
+          th= t_attr(:postal_code, Contact)
+          th colspan='3'
+
+      tbody
+        - @departments.each do |department|
+          tr
```

Comment:
with what advantage? In my view, this would spread the context, just for 6 lines of slim...
I see that as less readable